UserService/UserService/src/main/java/com/nt/user/microservice/contoller/UserController.java

```diff
@@ -0,0 +1,82 @@
+package com.nt.user.microservice.contoller;
+
+import com.nt.user.microservice.indto.LogInDTO;
+import com.nt.user.microservice.indto.UserInDTO;
+import com.nt.user.microservice.outdto.UserOutDTO;
+import com.nt.user.microservice.outdto.UserResponse;
+import com.nt.user.microservice.service.UserService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@CrossOrigin
+@RequestMapping("/api/users")
+public class UserController {
+
+  private static final Logger logger = LogManager.getLogger(UserController.class);
+
+  private final UserService userService;
+
+  public UserController(UserService userService) {
+    this.userService = userService;
+  }
+
+  @PostMapping("/register")
+  public ResponseEntity<UserResponse> registerUser(@Valid @RequestBody UserInDTO userInDTO) {
+    logger.info("Registering user with email: {}", userInDTO.getEmail());
+    System.out.println("Password="+userInDTO.getPassword());
+    String userAdded = userService.registerUser(userInDTO);
```

Comment:
Please don't use the short way as it is not a good practice. For line no. 41, 42 and 43 modify your service method it can not be done in the controller class method.